argocd/argoapp-smallstep.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: "smallstep"
+  namespace: "argocd"
+spec:
+  destination:
+    namespace: smallstep
+    server: https://kubernetes.default.svc
+  project: default
+  syncPolicy:
+    automated:
+      prune: true
+    syncOptions:
+      - CreateNamespace=true
+  source:
+    repoURL: https://github.com/advatys/smallstep-helm-charts.git
+    targetRevision: openshift
+    path: step-certificates
+    helm:
+      values: |
+        ca:
+          bootstrap:
+            # Add script snippets here to be executed after the step ca init has been run
```

Comment:
would be possible to add an example with ACME rather than OIDC ?

Reply:
will be done in another repo. Now added ACME installation by default into values.yaml